kuma/javascript/src/header/__snapshots__/header.test.js.snap

```diff
@@ -0,0 +1,258 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Header snapshot 1`] = `
+.emotion-32 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: row;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  border-top: 5px solid #83d0f2;
+}
+
+.emotion-0 {
+  display: block;
+  height: 48px;
+  width: 219px;
+  overflow: hidden;
+}
+
+.emotion-1 {
+  -webkit-box-flex: 0;
+  -webkit-flex-grow: 0;
+  -ms-flex-positive: 0;
+  flex-grow: 0;
+  -webkit-flex-shrink: 0;
+  -ms-flex-negative: 0;
+  flex-shrink: 0;
+  -webkit-flex-basis: 4px;
+  -ms-flex-preferred-size: 4px;
+  flex-basis: 4px;
+}
+
+.emotion-7 {
+  position: relative;
+  pointer-events: auto;
+}
+
+.emotion-5 {
+  font-size: 18px;
+  font-weight: bold;
+  padding: 5px 10px;
+}
+
+.emotion-5:hover {
+  background-color: #eee;
+}
+
+.emotion-3 {
+  font-size: 12px;
+  padding-left: 2px;
+}
+
+.emotion-21 {
+  -webkit-flex: 1 1 0px;
+  -ms-flex: 1 1 0px;
+  flex: 1 1 0px;
+}
+
+.emotion-25 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: row;
+  -ms-flex-direction: row;
+  flex-direction: row;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  border-bottom: solid black 1px;
+  max-width: 350px;
+  min-width: 120px;
+  -webkit-flex: 2 1 120px;
+  -ms-flex: 2 1 120px;
+  flex: 2 1 120px;
+}
+
+.emotion-23 {
+  vertical-align: -2px;
+  -webkit-flex: 0 0 20px;
+  -ms-flex: 0 0 20px;
+  flex: 0 0 20px;
+}
+
+.emotion-24 {
+  border-width: 0 !important;
+  font-size: 18px;
+  font-weight: bold !important;
+  padding-bottom: 0 !important;
+  -webkit-flex: 0 1 300px;
+  -ms-flex: 0 1 300px;
+  flex: 0 1 300px;
+}
+
+.emotion-26 {
+  -webkit-box-flex: 0;
+  -webkit-flex-grow: 0;
+  -ms-flex-positive: 0;
+  flex-grow: 0;
+  -webkit-flex-shrink: 0;
+  -ms-flex-negative: 0;
+  flex-shrink: 0;
+  -webkit-flex-basis: 15px;
+  -ms-flex-preferred-size: 15px;
+  flex-basis: 15px;
+}
+
+.emotion-29 {
+  font-size: 18px;
+  font-weight: bold;
+  color: black;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.emotion-29:hover {
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.emotion-28 {
+  vertical-align: -6px;
+}
+
+<div
+  className="emotion-32 emotion-33"
+>
+  <a
+    className="emotion-0"
+    href="http://localhost//"
```

Comment:
I didn't expect the double slash at the end of ``http://localhost//``.

Reply:
I didn't expect it either, but it seems harmless. The href is based on window.location, and there is a fallback for when window.location is not defined. I assumed it would not be defined in this node-based testing environment, since it is not a browser. But Jest does define it, but window.location.pathname is empty I guess? Or maybe there is an extra slash after the hostname?  Anyway: I'm pretty sure this is an artifact of the testing environment, but I'll double-check the code to make sure I'm not adding an extra slash.

Reply:
Oh; that's right. It is because there is supposed to be a locale between those slashes, but in the test environment window.location.pathname does not include a locale

Reply:
Fixed as a side effect of fixing your point about about just using the path and not the protocol and host.